lib/model/KnowledgePanelElement.dart

```diff
@@ -279,19 +305,14 @@ enum KnowledgePanelElementType {
   /// The description summarizes the knowledge panel.
   @JsonValue('text')
   TEXT,
```

Comment:
Please avoid to reformat your way by adding or removing empty lines.

Reply:
Actually that's what the dart formatter (dart format) generates.
This repo doesn't follow the dart rules?

Reply:
Oh keep it that way then. The rules change sometimes.